app/build.gradle

```diff
@@ -3,8 +3,8 @@ import com.google.gms.googleservices.GoogleServicesPlugin
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.1"
+    compileSdkVersion 30
+    buildToolsVersion "29.0.2"
```

Comment:
These values can be left as they are. We only have to change them if absolutely necessary as they would require modifications to the Docker image used for the automated CI builds.